visidata/loaders/xlsx.py

```diff
@@ -53,6 +53,61 @@ def iterload(self):
         for rownum in Progress(range(worksheet.nrows)):
             yield list(worksheet.cell(rownum, colnum).value for colnum in range(worksheet.ncols))
 
+            
+@VisiData.api
+def save_xlsx(vd, p, vs, *vsheets):
```

Comment:
This is not a full review, but please note that the api for this should have the form

```
def save_foo(vd, p, *sheets)
```

Reply:
Yup, done!